ozark/src/main/java/org/glassfish/ozark/cdi/OzarkCdiExtension.java

```diff
@@ -124,4 +137,45 @@ public static synchronized void addObservedEvent(Class<? extends MvcEvent> event
     public static synchronized boolean isEventObserved(Class<? extends MvcEvent> eventType) {
         return observedEvents == null ? false : observedEvents.contains(eventType);
     }
+
+    /**
+     * Registers given annotated types to be scanned during bean discovery
+     *
+     * @param beforeBean The BeforeBeanDiscovery
+     * @param beanManager The BeanManager
+     */
+    public void register(@Observes BeforeBeanDiscovery beforeBean, BeanManager beanManager) {
+
```

Comment:
It's a little bit confusing having two methods observing `BeforeBeanDiscovery`. Could we add the registration to `#beforeBeanDiscovery`?